BarrysPeanutsAkkaCascading/src/main/java/barryspeanuts/msg/CreditCard.java

```diff
@@ -0,0 +1,62 @@
+package barryspeanuts.msg;
+
+public class CreditCard {
+  private String nameOnCard;
+  private String creditCardNumber;
+  private Integer expirationMonth;
+  private Integer expirationYear;
+  private Integer securityCode;
+
+  public CreditCard(
+      String nameOnCard,
+      String creditCardNumber,
+      Integer expirationMonth,
+      Integer expirationYear,
+      Integer securityCode) {
```

Comment:
```suggestion
      int expirationMonth,
      int expirationYear,
      int securityCode) {   
```

Don't use primitive wrappers unless you have to (i.e. in generics, nullable types, or any other object-boxing need)